src/Spice86.Shared/Emulator/VM/Breakpoint/Serializable/SerializedBreakpointRange.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Spice86.Shared.Emulator.VM.Breakpoint.Serializable;
+/// <summary>
+/// Class for serializing memory range breakpoints
+/// </summary>
+public record SerializedBreakpointRange : SerializedBreakpoint {
```

Comment:
The record name is a bit misleading. I would call it SerializableBreakpointRange

Reply:
Done.